src/apps/containers/components/ContainerSearch.tsx

```diff
@@ -32,45 +36,73 @@ const useStyles = makeStyles((theme: Theme) =>
       background: "transparent",
       width: "100%",
       margin: theme.spacing(2),
-      padding: theme.spacing(2)
-    }
+      padding: theme.spacing(2),
+    },
+    flex: {
+      display: "flex",
+    },
   })
 );
 
 const ContainerSearch: React.FC<Props> = ({ title, onSearch, initialQ }) => {
   const classes = useStyles();
   const [q, setQ] = useState(initialQ);
+  const [state, setState] = React.useState({
+    displayVerified: false,
+  });
+  const handleChange = (event: any) => {
+    setState({ ...state, [event.target.name]: event.target.checked });
+    return event.target.checked ? onSearch(VERIFIED_SOURCES.CIEL) : onSearch();
+  };
   return (
     <Grid className={classes.searchContainer} item xs={12}>
-      <form
-        onSubmit={e => {
-          e.preventDefault();
-          onSearch(q);
-        }}
-      >
-        <Input
-          inputProps={{
-            className: classes.searchInput
+      <div className={classes.flex}>
+        <form
+          onSubmit={(e) => {
+            e.preventDefault();
+            onSearch(q);
           }}
-          onChange={e => setQ(e.target.value)}
-          value={q}
-          color="secondary"
-          type="search"
-          fullWidth
-          placeholder={`Search ${title}`}
-          data-testid="searchInput"
-          endAdornment={
-            <InputAdornment position="end">
-              <IconButton
-                onClick={() => onSearch(q)}
-                data-testid="searchButton"
-              >
-                <SearchIcon />
-              </IconButton>
-            </InputAdornment>
+        >
+          <Input
+            inputProps={{
+              className: classes.searchInput,
+            }}
+            onChange={(e) => setQ(e.target.value)}
+            value={q}
+            color="secondary"
+            type="search"
+            fullWidth
+            placeholder={`Search ${title}`}
+            data-testid="searchInput"
+            endAdornment={
+              <InputAdornment position="end">
+                <IconButton
+                  onClick={() => onSearch(q)}
+                  data-testid="searchButton"
+                >
+                  <SearchIcon />
+                </IconButton>
+              </InputAdornment>
+            }
+          />
+        </form>
+        <FormControlLabel
+          control={
+            <Switch
```

Comment:
Several things:

1. Does it make sense to have this toggle appear on *every* source screen? E.g., is there a use case for limiting things to "My verified sources" or should this just be limited to the public sources / dictionaries screen?
2. The label is relatively long and seems quite obtrusive and not fitting with Material design standards. Could we convert the label into, e.g., a tool tip or something less obtrusive? (Notice how Switches in [the Material docs](https://material.io/components/switches) are laid-out vertically rather than squashed on a line with other components as is done here. Maybe moving this to the line above the search box?
3. Should the state of this toggle persist as I change through views (i.e., when I go to "My Sources" and switch it on and then go to "Public Sources" shouldn't it still remain on?)
4. See screenshot below:
  
<img width="1230" alt="Screenshot showing toggle with Public Sources visible" src="https://user-images.githubusercontent.com/52504170/122245209-86ae0d00-ce93-11eb-83cb-1737b8b98347.png">

It seems weird to me that toggling this on is showing me what appear to be non-verified sources (I think those SNOMED sources are part of CIEL, but they aren't actually verified).

Reply:
@ibacher, I actually made changes with respect to design. @gracepotma, will you please add comments, how we can move forward?

Reply:
Thank you @ibacher  for the comment, I will update once I receive comments from Grace.

Reply:
1. I see what you're saying Ian but personally I've been finding that I create a number of fake dictionaries. It would be nice to easily filter these out, e.g. I notice the same thing happening for our main OHRI user when he's demoing and sharing his screen, his real work/real dictionaries are cluttered by practice/in-progress/not-prime-time-titled dictionaries. But, if it's easier to just apply this toggle to the Public tab, then yes, that's fine.

Reply:
2. @ibacher I agree this isn't quite the optimal design - if we wanted to design for possibly having more filter options in the future, we'd have an additional step/UI workflow for filtering - but for the sake of getting this out into the world more conceptually I am okay with this hack for now. (I know, slippery slope, so I do appreciate you raising this, as we'll need to protect the app from too many hacky UX practices.) The alternative that we played with in the sketches looked like this, do you prefer this layout? https://docs.google.com/presentation/d/1Q2yMyxTQrflQGc-O77KlGEXiPC_savon9ju9LDUaI5c/edit#slide=id.gda809a9155_1_38

Reply:
3. Yes, I agree it would be nice for the toggle state to persist. But this is a nice-to-have and could be added later, unless @suruchee this is easy to add to this PR.

Reply:
4. Agreed. @suruchee to work on this (turns out it's due to a hard coded filter to find things labeled with "CIEL" which those happen to have)

Reply:
@grace Re: 2. I personally prefer the layout with the toggle underneath the search, but I agree that it shouldn't be a blocker for this PR.

Reply:
Thanks @ibacher, yeah I actually prefer that layout too. @suruchee if that's easy to change go ahead, but if not please proceed :)

Reply:
@gracepotma Re: 1. I totally agree that a toggle to show my "real" dictionaries is very useful. I think I mostly just asked that since, for now, we only have a hard-coded list of sources that we're checking against so toggling the filter will *always* filter out all the sources on the My Sources page. Again, not a blocker, just seems like an... unexpected UX.

Reply:
Oh yeah good point @ibacher, you're totally right 🤦  Well, let's leave it for now, so that we don't have to worry about adding it back in later.